frontend/src/static/js/components/webstatus-stats-missing-one-impl-chart-panel.ts

```diff
@@ -112,6 +114,18 @@ export class WebstatusStatsMissingOneImplChartPanel extends WebstatusLineChartPa
     };
   }
 
+  updateFeatureListHref(featureList: MissingOneImplFeaturesList) {
+    if (!featureList.length) {
+      this.featureListHref = '';
+    }
+    let query = 'id:' + featureList[0].feature_id;
+    for (let i = 1; i < featureList.length; i++) {
+      query += ' OR id:' + featureList[i].feature_id;
+    }
+
+    this.featureListHref = formatOverviewPageUrl({search: '?q=' + query});
```

Comment:
I know the urls.test.ts uses `search` directly but the non test code uses the option object to set `q`

https://github.com/GoogleChrome/webstatus.dev/blob/2b32117992b16950c5e27193253e4c50233cd9b4/frontend/src/static/js/components/webstatus-sidebar-menu.ts#L235-L236

Which leverages this:

https://github.com/GoogleChrome/webstatus.dev/blob/2b32117992b16950c5e27193253e4c50233cd9b4/frontend/src/static/js/utils/urls.ts#L74-L83

This prevents us from manually managing the query parameters.

Reply:
Done! Thanks!